content/en/synthetics/guide/monitor-https-redirection.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Monitor your HTTP requests are redirected into HTTPS
+kind: guide
+further_reading:
+    - link: '/synthetics/api_tests/http_tests'
+      tag: 'Documentation'
+      text: 'Create a HTTP Test'
+
+---
+
+## Overview
+
+Monitoring your HTTP traffic is redirected into HTTPS is critical in ensuring your users' connections are encrypted with your API endpoints and your application.
+
+### Monitor your HTTPS redirection
+
+Depending on your setup, you can identify the redirect to HTTPS in the generated **Response Preview** tab under Headers as `location` or in the **Body** as `"https:"===window.location.protocol`.
+
+To monitor the redirection of your HTTP traffic into HTTPS:
+
+1. Create an HTTP test and [define the request][1].
+2. Click **Test URL**. The response preview generates a **Request Preview** and **Response Preview**.
+3. Add an assertion about the redirection to HTTPS.
+    - Define an assertion on the `location` header. For example, under **Headers**, the `location` header for `http://datadoghq.com` is `https://datadoghq.com`.
+    
+    {{< img src="synthetics/guide/monitor-https-redirections/location-header-https.png" alt="Location header in the response preview" style="width:100%;" >}}
+    - Optionally, select `body` `contains` and paste `"https:"===window.location.protocol` in the text field. 
```

Comment:
it is really either assert on header "location" OR assert on the body.
So I feel "optionally" is not the right word

Reply:
```suggestion
    - Alternatively, define an assertion on the response body by clicking **+ New Assertion**. Select `body` `contains` and paste `"https:"===window.location.protocol` in the text field. 
```